ironic.conf.j2

```diff
@@ -71,23 +117,16 @@ pxe_config_template = $pybasedir/drivers/modules/ipxe_config.template
 tftp_master_path = /shared/tftpboot
 tftp_root = /shared/tftpboot
 uefi_pxe_config_template = $pybasedir/drivers/modules/ipxe_config.template
+pxe_append_params = nofb nomodeset vga=normal ipa-insecure=1
```

Comment:
Oh, nearly forgot! Redfish virtual media does not use (because it does not use PXE), you need to update this option: https://opendev.org/openstack/ironic/src/branch/master/ironic/conf/redfish.py#L66 (I intend to make host validation work with virtual media, but it's not quite there).

Reply:
do we need to set ipa-inspection-callback-url and possibly ipa-api-url here ?

Reply:
AFAIR no